solution.txt

```diff
@@ -203,13 +186,8 @@ Exercise 15:
 */
 
 // Complete Exercise 15 below...
-
 let total = 0;
+numArrays.forEach(array => {array.forEach((element) => {total += element})});
```

Comment:
nice! the instructions specifically say to use `forEach()`, but if they didn't, the other obvious array method would be `reduce()`. You imitate the functionality of `reduce()` with `forEach()` really well here.